packages/lib/time.ts

```diff
@@ -110,6 +110,13 @@ class Time {
 		return m.isValid() ? m.toDate() : defaultValue;
 	}
 
+	public anythingToMs(o: any, defaultValue: Date = null) {
+		if (o && o.toDate) return o.toDate();
+		if (!o) return defaultValue;
+		const m = moment(o);
```

Comment:
Can moment throw an error if the input is not valid?

Reply:
I'm having trouble finding confirmation for this in the docs. So I tested with a variety of inputs and it looks like moment just returns null if it gets anything invalid. This should be safe as far as I can tell.

Reply:
Ok looks good then, thanks for checking!